myenv/Scripts/activate

```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
These files where added because you probably ran `git add .` - which adds everything that is new to git - and also because the default name that we have on `.gitignore` for virtual environments is `venv` and you named it `myenv`.
Could you delete/rename that `myenv` to `venv`. I am asking that because it does not seem that it would scale well, you putting this on the `.gitignore` file. What you can do next is instead of performing `git add .` perform `git add <path/to/file>` for each of the files you which to commit :)

Makes sense @vj-codes ?

Reply:
Thanks @isabelcosta .I didn't know about using git add and specific path and it'll make such a huge difference.Glad learnt it now! 
I have removed all those files now , please have a look again.

Reply:
I am super glad that you learned something here 🎉 🤗  @vj-codes